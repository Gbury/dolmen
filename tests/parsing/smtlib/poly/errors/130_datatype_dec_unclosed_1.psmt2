(set-logic ALL)
(declare-datatype foo (par (a) ((A)) _))
