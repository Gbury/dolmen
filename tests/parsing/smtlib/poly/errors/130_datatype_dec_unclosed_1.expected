File "tests/parsing/smtlib/poly/errors/130_datatype_dec_unclosed_1.psmt2", line 2, character 37-38:
Error while parsing a datatype declaration, read the reserved word '_',
      but expected a closing parenthesis.
